pkg/symlink/fs.go

```diff
@@ -1,10 +1,26 @@
+/*
+   Copyright The containerd Authors.
```

Comment:
And also Moby authors

Reply:
moby repo does not seem to have a copyright header like this. will add another line after line 21 to indicate where we picked this up from,